app/views/records/file_manager.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1>File Manager</h1>
 <ul class="breadcrumb">
-  <li><%= link_to @record.title.to_sentence, solr_document_path(id: "id-#{@record.id}") %></li>
+  <li><%= link_to @record.title.to_sentence, solr_document_path(id: "#{@record.id}") %></li>
```

Comment:
unnecessary string interpolation here?

Reply:
No, I could make it @record.id.to_s though. That better?

Reply:
Oh, nevermind, you're right.